<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_ps_file</class>
 <widget class="QDialog" name="new_ps_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="w">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度(inch)</string>
   </property>
  </widget>
  <widget class="QLabel" name="h">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>69</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>高度(inch)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="w_input">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>38</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入数字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="h_input">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>68</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入数字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;按TAB键可在各选项间快速切换&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PS文件名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ps_fname">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>.ps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;i&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>68</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;i&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>130</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="J_Layout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="R_Layout"/>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#868686;&quot;&gt;-J -R 任一项留空将默认都使用-JX1/1 -R0/1/0/1：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>93</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bexit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="w_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度(cm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>69</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>高度(cm)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="h_input_cm">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>68</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入数字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="w_input_cm">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>38</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入数字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>68</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;cm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;cm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="A4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>使用A4纸</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>A4</tabstop>
  <tabstop>w_input_cm</tabstop>
  <tabstop>h_input_cm</tabstop>
  <tabstop>w_input</tabstop>
  <tabstop>h_input</tabstop>
  <tabstop>ps_fname</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>bexit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
